system-variables.md

```diff
@@ -378,21 +378,15 @@ mysql> SELECT * FROM t1;
 
 ### `tidb_enable_async_commit` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
 
-> **警告：**
->
-> 当前该功能为实验特性，不建议在生产环境中使用。目前存在已知问题有：
->
-> + 暂时与 [TiCDC](/ticdc/ticdc-overview.md) 不兼容，可能导致 TiCDC 运行不正常。
-> + 暂时与 [Compaction Filter](/tikv-configuration-file.md#enable-compaction-filter-从-v500-rc-版本开始引入) 不兼容，共同使用时有小概率发生写丢失。
-> + 本特性与 TiDB Binlog 不兼容，开启 TiDB Binlog 时本配置将不生效。
-
 - 作用域：SESSION | GLOBAL
-- 默认值：OFF
+- 默认值：对于新创建的集群，v5.0.0-rc 版本默认值为 OFF，自 v5.0.0 GA 版本起默认值为 ON。对于升级到 v5.0.0 GA 版本的集群，如果升级前是 v5.0.0-rc 版本，升级不改变该变量的值；如果升级前是 v4.0 及之前版本，升级后默认值为 OFF。
```

Comment:
@qiancai @sticnarf 
+ Let's remove the version-specific information in master and only retain the latest applicable default value. If this PR is for 5.0 GA, looks like it should be "ON"?
+ v5.0.0 GA --> v5.0 GA
+ For v5.0.0-rc, I think we should unify the spelling to v5.0 RC because there is no v5.0.1-rc or v5.0.2-rc. A release candidate version is only for once? @sticnarf

Reply:
For v5.0master:

Reply:
@TomShawn and @sticnarf, PTAL, thanks~

For v5.0: 
```suggestion
- 默认值：对于新创建的集群，v5.0 RC 版本默认值为 OFF，自 v5.0 GA 版本起默认值为 ON。对于升级到 v5.0 GA 版本的集群，如果升级前是 v5.0 RC 版本，升级不改变该变量的值；如果升级前是 v4.0 及之前版本，升级后默认值为 OFF。
```
For master:
```suggestion
- 默认值：对于新创建的集群，默认值为 ON。对于升级版本的集群，如果升级前是 v5.0 版本，升级不改变该变量的值；如果升级前是 v4.0 及之前版本，升级后默认值为 OFF。
```

Reply:
LGTM